api/users_test.go

```diff
@@ -0,0 +1,145 @@
+package api
+
+import (
+	"bytes"
+	"io"
+	"net/http"
+	"strings"
+	"testing"
+
+	qt "github.com/frankban/quicktest"
+)
+
+func Test_registerHandler(t *testing.T) {
```

Comment:
idiomatic would be TestRegisterHandler but i guess you have a reason to deviate

Reply:
I thought that if the method to test is private the test function must follow that format, but reading again about this I was wrong :(